Update cross-tenant-mailbox-migration.md

zouma83 opened the pull request:
### How do I know this worked?
=> testing relationship^and endpoint before/after creating at list one object in the target tenant
;)
FM

zouma83 committed:
Update cross-tenant-mailbox-migration.md

### How do I know this worked?
=> testing relationship^and endpoint before/after creating at list one object in the target tenant
;)
FM

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

JohanFreelancer9 reviewed (commented):
@yogkumgit - Could you please request a review from the document author or manager before I do a copy review. I am not at all sure about this.

yogkumgit commented:
@kelleyvice-msft Please help. Thanks

zouma83 committed:
Merge branch 'public' into patch-1

zouma83 commented:
hi all

tested and used twice during two migrations with the actual manual
> Test-MigrationServerAvailability -Endpoint "[the name of your cross-tenant migration endpoint]" -TestMailbox "[email address of a source mailbox that is part of your migration scope]"

nok error, even before/after creating target object(s)
cf. command's documentation Test-MigrationServerAvailability seems not working with -Endpoint and -TestMailbox parameters together


so for checking the source & target organisation's preparations:
target tenant:
> Test-MigrationServerAvailability -Endpoint "[the name of your cross-tenant migration endpoint]"
> Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability

source tenant:
> Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability

ok, informations indicate ready for mailboxes migrations for both organisations (source & target)
these commands came from the previous manual's version (tested and used several times too)

thanks

zouma83 committed:
Update cross-tenant-mailbox-migration.md

Test-MigrationServerAvailability can't run with both "-Endpoint" & "-TestMailbox" parameters together

zouma83 committed:
Merge branch 'public' into patch-1

JohanFreelancer9 commented on microsoft-365/enterprise/cross-tenant-mailbox-migration.md:
```diff
@@ -206,7 +206,17 @@ To obtain the tenant ID of a subscription, sign in to the [Microsoft 365 admin c
    > [!NOTE]
-   > Test-MigrationServerAvailability -Endpoint "[the name of your cross-tenant migration endpoint]" -TestMailbox "[email address of a source mailbox that is part of your migration scope]"
+   >
+   > - target tenant:
+   > 
+   > Test-MigrationServerAvailability -Endpoint "[the name of your cross-tenant migration endpoint]"
+   >
+   > Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability
+   >
+   > - source tenant:
+   > 
+   > Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability
+   
```
```suggestion
   > - Target tenant:
   > 
   > Test-MigrationServerAvailability -Endpoint "[the name of your cross-tenant migration endpoint]"
   >
   > Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability
   >
   > - Source tenant:
   > 
   > Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability 
```

JohanFreelancer9 reviewed (changes requested):
Copy review - @zouma83, kindly commit the minor suggested changes. Many thanks.

zouma83 committed:
Update microsoft-365/enterprise/cross-tenant-mailbox-migration.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

zouma83 committed:
Merge branch 'public' into patch-1

zouma83 committed:
Merge branch 'public' into patch-1

zouma83 committed:
Merge branch 'public' into patch-1

yogkumgit commented:
@kelleyvice-msft Please merge this PR. Thanks

zouma83 committed:
Merge branch 'public' into patch-1

zouma83 committed:
Merge branch 'public' into patch-1

zouma83 committed:
Merge branch 'public' into patch-1

zouma83 committed:
Merge branch 'public' into patch-1

zouma83 committed:
Merge branch 'public' into patch-1

kelleyvice-msft committed:
Merge branch 'public' into patch-1

kelleyvice-msft committed:
Merge branch 'public' into patch-1

kelleyvice-msft committed:
Merge branch 'public' into patch-1
